### PR TITLE
Property 'setState' error rectified.

### PR DESCRIPTION
**Pull request by Suyash2810**
Property 'setState' does not exist on type 'InnerMe' error in TestMe.tsx rectified.

### CODE REVIEW

**Comment by ahtee** [comment]
can you please rebase to `master`? looks like something may have also gotten force pushed so expect merge conflicts...?

**Comment by Suyash2810** [comment]
> can you please rebase to `master`? looks like something may have also gotten force pushed so expect merge conflicts...?

Everything seems to be up to date and I think there are no merge conflicts.

File: bonus-jest/demo/src/TestMe.tsx [hunk 1]
```diff
@@ -17,21 +17,25 @@ export const TestMe = (props: TestMeProps) => {
...
+    throw new Error("Method not implemented.");
+  }
   state = {
     clicked: false
   };
 
   onClick = () => {
     this.setState({ clicked: true });
   };
+  props: any;
```
Comment (ahtee) [inline bonus-jest/demo/src/TestMe.tsx]: not familiar for why this is here

File: bonus-jest/demo/src/TestMe.tsx [hunk 2]
```diff
@@ -17,21 +17,25 @@ export const TestMe = (props: TestMeProps) => {
...
 
   render() {
     return !this.state.clicked ? (
       <div onClick={this.onClick} id="innerMe">
         Hello {this.props.name}, Click Me
       </div>
     ) : (
-      <div id="innerMe">Clicked</div>
-    );
+        <div id="innerMe">Clicked</div>
```
Comment (ahtee) [inline bonus-jest/demo/src/TestMe.tsx]: remove this spacing

File: bonus-jest/demo/src/TestMe.tsx (hunk 1 above)
Comment (Suyash2810) [reply bonus-jest/demo/src/TestMe.tsx]: There was a type error and using this rectified it.

File: bonus-jest/demo/src/TestMe.tsx (hunk 2 above)
Comment (Suyash2810) [reply bonus-jest/demo/src/TestMe.tsx]: The code editor is automatically providing spaces in this part. Is it necessary?

Comment (ahtee) [reply bonus-jest/demo/src/TestMe.tsx]: Probably resulting from the ternary? In that case it is fine I overlooked the ternary

Comment (Suyash2810) [reply bonus-jest/demo/src/TestMe.tsx]: So is it okay to let it be that way?

Comment (ahtee) [reply bonus-jest/demo/src/TestMe.tsx]: > So is it okay to let it be that way?

Yes

File: bonus-jest/demo/src/TestMe.tsx (hunk 1 above)
Comment (Suyash2810) [reply bonus-jest/demo/src/TestMe.tsx]: Are there any more changes that are required to be made or is it good to go?

**Comment by ecraig12345** [comment]
`InnerMe` extends `React.Component`, which means that `setState` and `props` will automatically be available. The only reason I can think of that `setState` and `props` wouldn't be available is if you didn't run `npm install` before `npm start`.

**Comment by ecraig12345** [comment]
Closed this PR since I don't think it's the right fix. Feel free to open an issue and tag me if you'd like help debugging what's going wrong.

**Pull request closed without merging**